Mathlib/MeasureTheory/Group/GeometryOfNumbers.lean

```diff
@@ -83,4 +85,51 @@ theorem exists_ne_zero_mem_lattice_of_measure_mul_two_pow_lt_measure [NormedAddC
   refine' h_conv hw (h_symm _ hv) _ _ _ <;> norm_num
 #align measure_theory.exists_ne_zero_mem_lattice_of_measure_mul_two_pow_lt_measure MeasureTheory.exists_ne_zero_mem_lattice_of_measure_mul_two_pow_lt_measure
 
+variable  (h_cpt : IsCompact s) (h_mes : μ s ≠ 0) [DiscreteTopology L] [Nontrivial E]
+
+/-- The version of Minkowski Convex Body Theorem where we assume in addition that `s` is compact.
+In this case, the strict inequality can be replaced by a weak inequality. -/
```

Comment:
This docstring is not really self-contained as it requires knowing what the Minkowski theorem is. Could you rather make it self-contained by modelling it on the docstring of the previous theorem?

Reply:
I changed that and added some precision about the difference between the two theorems.